ports/sjpeg/portfile.cmake

```
@@ -0,0 +1,25 @@
+if(VCPKG_TARGET_IS_WINDOWS)
+    vcpkg_fail_port_install(ON_LIBRARY_LINKAGE "dynamic")
```

```suggestion
    vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
```
We prefer to use this to express a port that only supports static build. https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/vcpkg_check_linkage.md

---

Applied your suggestion in 906e4bade39fbcdceb69474e2ad7d67036e6ac01.